tests/unit/test_client.py

```diff
@@ -1106,3 +1106,61 @@ def test_verify_takes_precedence_over_scoped_config(self):
                               scoped_config={'ca_bundle': 'mycerts.pem'})
         call_kwargs = self.endpoint_creator.create_endpoint.call_args[1]
         self.assertEqual(call_kwargs['verify'], 'verify.pem')
+
+
+class TestConfig(unittest.TestCase):
```

Comment:
There was one already. I added a check though to make sure that there is no default set prior to sending the config object